source/Loggly/Loggly.nuspec

```diff
@@ -1,18 +1,38 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <metadata>
-        <id>loggly-csharp</id>
+        <id>loggly-csharp-netcore</id>
         <version>$version$</version>
         <title>loggly-csharp</title>
         <authors>neutmute</authors>
         <projectUrl>https://github.com/neutmute/loggly-csharp</projectUrl>
         <iconUrl>https://raw.githubusercontent.com/neutmute/loggly-csharp/master/SolutionItems/loggly.png</iconUrl>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
-        <description>$description$</description>
+        <description>A .NET client for loggly. Supporting Https, Syslog UDP and encrypted Syslog TCP transports.</description>
         <language>en-US</language>
         <tags>loggly</tags>
+        <dependencies>
+            <group targetFramework="net45">
+                <dependency id="loggly-csharp-config-netcore" version="$version$" />
+                <dependency id="Newtonsoft.Json" version="6.0.8" />
+            </group>
+            <group targetFramework=".NETStandard1.5">
+                <dependency id="System.Diagnostics.Process" version="4.1.0" />
+                <dependency id="System.Net.Http" version="4.1.0" />
+                <dependency id="System.Net.NameResolution" version="4.0.0" />
+                <dependency id="System.Net.Requests" version="4.0.11" />
+                <dependency id="System.Net.Security" version="4.0.0" />
+                <dependency id="System.Net.Sockets" version="4.1.0" />
+                <dependency id="System.Diagnostics.Tracing" version="4.1.0" />
+                <dependency id="loggly-csharp-config-netcore" version="$version$" />
```

Comment:
The versions of these two packages should be the same for this to work.  This should be the case when building using `build.ps1`.